test/run-openshift

```diff
@@ -104,15 +121,13 @@ function test_mysql_s2i() {
                           --env MYSQL_PASSWORD=testnormal
 
   ct_os_wait_pod_ready "${service_name}" 60
-  check_mysql_os_service_connection "${image_name}" "${service_name}" testo testo
+  check_mysql_os_service_connection "${image_name}" "${service_name}" testo testo 120
 
   ct_os_delete_project
 }
 
-ct_os_install_in_centos
 ct_os_cluster_up
 test_mysql_pure_image ${IMAGE_NAME}
 test_mysql_template ${IMAGE_NAME}
-test_mysql_s2i ${IMAGE_NAME} "https://github.com/hhorak/mariadb-container.git#s2i-support-3" test/test-app
-
-
+# TODO: Can we make the build against examples inside the same PR?
+test_mysql_s2i ${IMAGE_NAME} "https://github.com/sclorg/mariadb-container.git" test/test-app
```

Comment:
You could try omitting the `.git` from url.

Reply:
This should be working with `.git` as all of oc's help messages have it in there but I can try it out